README.md

```diff
@@ -83,6 +90,43 @@ DeployGate.registerCallback(new DeployGateCallback() {
 
 For the more details, pelase refer to [DeployGate Docs - Android SDK](https://docs.deploygate.com/docs/developer-tools/android-sdk/)
 
+### Capture feature
+
+Workspaces, which have activated the Capture feature, can storee screenshots, device states and Logcat at DeployGate. The Capture feature is enabled by default.
+
+```java
+import com.deploygate.sdk.DeployGate;
+import com.deploygate.sdk.DeployGateSdkConfiguration;
+
+DeployGateSdkConfiguration config =
+                new DeployGateSdkConfiguration().
+                   setCaptureEnabled(true). // Defaults to true
+                   build();
```

Comment:
We need to use the Builder class for making a configuration with builder pattern.

```suggestion
DeployGateSdkConfiguration config = 
    new DeployGateSdkConfiguration.Builder()
        .setCaptureEnabled(true) // Defaults to true
        .build();
```

Reply:
Oh, good catch